src/common/firstLettersOfWords.ts

```diff
@@ -33,7 +35,12 @@ export function firstLettersOfWords(
   const pattern = words.match(hasUppercaseLettersPattern)
     ? uppercasePattern
     : lowercasePattern;
-  const letters = Array.from(words.matchAll(pattern), (m) => m[1]).join('');
+  let letters = Array.from(words.matchAll(pattern), (m) => m[1]).join('');
+
+  letters = isEmpty(letters)
+    ? Array.from(words.matchAll(lowercasePattern), (m) => m[1]).join('')
+    : letters;
```

Comment:
```suggestion
  letters = letters || Array.from(words.matchAll(lowercasePattern), (m) => m[1]).join('');
```

Reply:
Better yet would be to abstract that function
```tsx
const matchAndMerge = (pattern: RegExp, str: string, group = 1) =>
  Array.from(str.matchAll(pattern), (m) => m[group]).join('');
```
Then it usage looks even better
```tsx
  const letters =
    matchAndMerge(pattern, words) || matchAndMerge(lowercasePattern, words);
```